### PR TITLE
Switch to go modules

### PR DESCRIPTION
**Pull request by jlegrone**
Closes https://github.com/cnabio/cnab-go/issues/156.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
 We have [good first issues][good-first-issue] for new contributors and [help wanted][help-wanted] issues for our other contributors.
 
-* `good first issue` has extra information to help you make your first contribution.
```
Comment (glyn) [inline CONTRIBUTING.md]: There seem to be a lot of changes in common with the import renaming PR. Probably best to merge that one first and then rebase this?

Comment (jlegrone) [reply CONTRIBUTING.md]: Yes, as noted in the PR description. Technically this could be merged separately but I wanted to see that tests still worked.

**Comment by jlegrone** [comment]
This will supercede https://github.com/cnabio/cnab-go/pull/52 if merged.

**Pull request merged**